src/Moose-Algos-Graph/MalHitsNode.class.st

```diff
@@ -35,13 +35,13 @@ MalHitsNode >> label [
 ]
 
 { #category : #printing }
-MalHitsNode >> printString [
-
-	^ 	'(',
-		self model printString,
-		' ',
-		(self auth printShowingDecimalPlaces: 2),
-		' ',
-		(self hub printShowingDecimalPlaces: 2),
-		') '
+MalHitsNode >> printOn: aStream [
+	aStream nextPut: $(.
+	self model printOn: aStream.
+	aStream nextPut: Character space.
+	self auth printOn: aStream showingDecimalPlaces: 2.
+	aStream nextPut: Character space.
+	self hub printOn: aStream showingDecimalPlaces: 2.
+	aStream nextPutAll: ') '.
```

Comment:
This is going to change the former behavior.
I'm removing the space at the end (from tests too) because I also think it's better to avoid it.
Let me know if this is ok with you.

Reply:
The printOn: should be used only for development purposes, so it is ok to change. Clients should have to add a space to fix the issue. But they can also put a CR if they want